subsys/net/lib/nrf_cloud/src/nrf_cloud_credentials.c

```diff
@@ -192,27 +243,50 @@ int nrf_cloud_credentials_check(struct nrf_cloud_credentials_status *const cs)
 
 	ret = cred_exists(cs->sec_tag, CA_CERT, &exists);
 	if (ret < 0) {
+		LOG_ERR("Error checking CA exists");
 		return -EIO;
 	}
 	cs->ca = exists;
+	if (exists) {
+		ret = cred_size_get(cs->sec_tag, CA_CERT, &cs->ca_size);
+		if (ret < 0) {
+			LOG_ERR("Error checking CA size");
+			return -EIO;
+		}
+		/* These flags are approximate and only useful for logging to help diagnose
+		 * possible provisioning mistakes.
+		 */
+		cs->ca_combined = (cs->ca_size > CONFIG_NRF_CLOUD_COMBINED_CA_CERT_SIZE_THRESHOLD);
```

Comment:
can the combined flag be removed?
if both coap and aws flags are set then that indicates it is combined.

```
if (size > combined_min) {
	coap = 1
	aws = 1
} else if (size <= coap_max) {
	coap = 1
	aws = 0
} else if (size > coap_max) {
	coap = 0
	aws = 1
}
```